package.json

```diff
@@ -34,12 +34,17 @@
   "author": "HomeOffice",
   "dependencies": {
     "browserify": "^12.0.1",
+    "express": "^4.14.0",
     "hof-bootstrap": "^6.0.0",
+    "hof-controllers": "^4.0.1",
     "hof-frontend-toolkit": "^1.0.0",
+    "hof-model": "^2.1.0",
     "hof-transpiler": "^0.1.0",
     "jquery": "^3.1.0",
+    "lodash": "^4.16.3",
     "node-sass": "^3.10.0",
```

Comment:
That is fine, I wouldn't have mentioned it but there seem to be some deps updated.